oat-sa/tao-core
# Feature/aut 3633/task queue solar design

related to https://oat-sa.atlassian.net/browse/AUT-3633

### Description 

TaskQueue component is updated to Solar design with this PR.

 - header bar badge button sizing and style is updated
 - list of tasks is updated
 - deatiled view of a task is updated

<img width="469" alt="Screenshot 2024-05-24 at 17 06 45" src="https://github.com/oat-sa/tao-core/assets/10635482/50d04a51-503c-47d2-aadf-566e8baf4d67">
<img width="843" alt="Screenshot 2024-05-24 at 17 07 10" src="https://github.com/oat-sa/tao-core/assets/10635482/31145eed-d980-4a7c-84a3-a80d3e3c2ed3">

### Details
To activate the feature, you need the feature flag FEATURE_FLAG_SOLAR_DESIGN_ENABLED.

`php index.php 'oat\tao\scripts\tools\FeatureFlag\FeatureFlagTool' -s FEATURE_FLAG_SOLAR_DESIGN_ENABLED -v true`
Once activated, the CSS class solar-design is added to the body element of the page. Then, the stylesheet automatically applies.

### How to test
 - check out the branch: git checkout -t origin/feature/AUT-3633/task-queue-solar-design
 - activate the feature flag: php index.php 'oat\tao\scripts\tools\FeatureFlag\FeatureFlagTool' -s FEATURE_FLAG_SOLAR_DESIGN_ENABLED -v true
 - log in to TAO backoffice
 - create some task like delivery publishing, check the task queue
